### DISCUSSION TITLE
No directory recursion on search?

### DISCUSSION THREAD

**Discussion by flow2k** [community/community#21865] [New to GitHub]
<p>After reading the doucumentation, I know I can do</p>
<p><code>path:/</code> to search the root directory without recursing into subdirectories. But how can I search an arbitrary directory without recursing into subdirectories?</p>
<p>For example, I only want to search the <code>abc</code> directory. I tried <code>path:/abc/</code> but it recurses into subdirectories under<code>abc</code> as well. </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You should be able to use <code>path:abc</code> as the specifier.</p>
